chain/actions/src/action_queue.rs

```diff
@@ -2,34 +2,33 @@
 //!
 //! The [ActionQueue] acts as a MPSC queue of [Actions](hopr_chain_types::actions::Action) which are executed one-by-one
 //! as they are being popped up from the queue by a runner task.
-use async_channel::{bounded, Receiver, Sender};
-use async_trait::async_trait;
-use futures::future::Either;
-use futures::{pin_mut, FutureExt, StreamExt};
-use serde::{Deserialize, Serialize};
 use std::fmt::{Display, Formatter};
 use std::future::Future;
 use std::pin::Pin;
 use std::sync::Arc;
 use std::time::Duration;
-use tracing::{debug, error, info, trace, warn};
 
+use async_channel::{bounded, Receiver, Sender};
+use async_trait::async_trait;
+use futures::future::Either;
+use futures::{pin_mut, FutureExt, StreamExt};
 use hopr_async_runtime::prelude::spawn;
```

Comment:
```suggestion
use futures::{pin_mut, FutureExt, StreamExt};

use hopr_async_runtime::prelude::spawn;
```

Reply:
Workspace dependencies should be separated into their own paragraph, just like crate exclusive dependencies.

Reply:
There is no such option in rustfmt.